silhouette/app/com/mohiva/play/silhouette/impl/providers/oauth2/GoogleProvider.scala

```diff
@@ -92,6 +92,7 @@ class GoogleProfileParser extends SocialProfileParser[JsValue, CommonSocialProfi
     val lastName = (json \ "name" \ "familyName").asOpt[String]
     val fullName = (json \ "displayName").asOpt[String]
     val avatarURL = (json \ "image" \ "url").asOpt[String]
+    val isDefaultAvatar = (json \ "image" \ "isDefault").asOpt[Boolean].getOrElse(false)
```

Comment:
Opinionated default, and is we cannot find the flag in the response, fallback to use the URL as the user's profile picture.